### ISSUE TITLE
[Suggestion] Subtitles

### ISSUE THREAD

**Issue by Ch1216** [MehVahdJukaar/Supplementaries#70]
When throwing a brick subtitles says "Snowball Flies"
When exiting out of a blackboard subtitles say 'Cartographer Working"
When springs are activated subtitles say "Piston moves"
When putting something into a Hourglass subtitles say "Item frame fills"

These subtitles are clearly wrong, There should be new subtitles for these actions

Also when applying honey to a cake it says "Block Placed" which could be technically correct if you count liquid honey as a block

**Comment by MehVahdJukaar** [comment]
yeah every single sound that the mod use is a vanilla one so of course it plays its subtitles. I'm planning to make custom sounds for many of these so they will come (I'll need to find somebody that can do them/get them first tho)
